discord_bot.py

```diff
@@ -1,9 +1,44 @@
 import logging
 
+import discord
 from discord.ext import commands
 
+from cogs.poll import get_new_description, string_description_to_dictionary
+
 
 class HuskieBot(commands.Bot):
 
     async def on_ready(self):
         logging.info('Huskie Bot Online')
+
+    async def on_raw_reaction_add(self, payload):
+        """ This method is triggered when there is a reaction added to any message. For now it only works for messages
+        that have a Poll in the embed footer.
+
+        :param payload: RawReactionActionEvent
+        :return: None
+        """
+        updated_message = await self.get_channel(payload.channel_id).fetch_message(payload.message_id)
+        description_dict = string_description_to_dictionary(updated_message.embeds[0].description)
+        if "Poll" in updated_message.embeds[0].footer.text:
```

Comment:
Can we make this check a little more robust. Other bots are in our channel and other users may post something that could contain an embed with the word "poll". could we generate a unique key?

Reply:
We can generate a key for polls in general. Something that any poll will be generated with that will always be the same for every poll. Since when the poll is over it deletes the footer we shouldn't run into any problems really.